src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -107,7 +107,7 @@
             class="tc_submenuitem tc_submenuitem_cluster_inventory"
             *ngIf="permissions.hosts.read">
           <a i18n
-             routerLink="/inventory">Inventory</a>
+             routerLink="/inventory">Physical disks</a>
         </li>
```

Comment:
From tracker, I see to rename this as Either Disks or Storage. So I think there should be an agreement here.

Reply:
I just asked in Daily standup meeting with @epuertat  and @avanthakkar  mentioned we should go with Physical disks. Since it was Sage who suggested. I don't mind changing. Let me know how should I go with the agreement.

Reply:
Well, I don't have any objections over here. Since @epuertat suggested it, lets wait for him. But if we are agree on `Disks` then we should make change over here too https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/inventory/inventory.component.html#L3

Reply:
ok ! Let him confirm and then will do the same for both the files. 
Also does it mean we need to change the name of component too?

Reply:
@epuertat  Could you please confirm if I could go ahead with 'Physical Disks' instead of Inventory?

Reply:
Just Confirmed with @epuertat  in daily meetup, will go  ahead with "Physical Disks".

Reply:
@navinbarnwal @nizamial09 yup, let's go with Physical Disks! I usually prefer the shorter, but Sage and others suggested this, so let's go with it.